spyderlib/config.py

```diff
@@ -169,6 +169,7 @@ def is_ubuntu():
               'animated_docks': True,
               'prompt_on_exit': False,
               'panes_locked': True,
+              'interface_language': load_lang_conf(),

```

Comment:
I think we should leave it as it is, otherwise we would have to made unneded changes either to how the preferences are loaded or do an extra check in spyder.y to see if this is the first time it runs.... here it is just one line of code...